### ISSUE TITLE
Screen Night Light 

### ISSUE THREAD

**Issue by paiyayay** [Jas-SinghFSU/HyprPanel#86]
**Is your feature request related to a problem? Please describe.**
Night light is comforting at night.

**Describe the solution you'd like**
A button to toggle nightlight, and even better if we could set time intervals for it to open automatically.

**Describe alternatives you've considered**
[wl-gammarelay-rs](https://github.com/MaxVerevkin/wl-gammarelay-rs) works pretty well for this. It can be used to change the display temperature.

```
busctl --user set-property rs.wl-gammarelay / rs.wl.gammarelay Temperature q 3500
# ex. Value 6500 being the default, and 3500 being the nightlight temperature.
```
**Additional context**
none

**Comment by matavach** [comment]
This won't be too hard to add.

**Comment by Jas-SinghFSU** [comment]
I think this is a great tool but would the ability to add custom shortcuts/scripts to the dashboard not satisfy your requirement?

**Comment by paiyayay** [comment]
yes ofc I already implemented it. But I thought it would be a nice feature to have. Like any other extra feature

**Comment by Jas-SinghFSU** [comment]
I think it's neat too; I'll have to think about this one. This isn't a niche feature so I can see it being integrated into the panel.

**Comment by rubiin** [comment]
hyprland now has with its own nightlight shader https://github.com/hyprwm/hyprsunset
the downside for now is you have to be on the git version of hyprland. This can be integrated with a custom module

**Comment by rubiin** [comment]
another option for the time being is hyprshade which i am using right now

**Comment by paiyayay** [comment]
> another option for the time being is hyprshade which i am using right now

that's nice! let's wait for it to come out to stable then implement it.

**Comment by Jas-SinghFSU** [comment]
Added a bar module to toggle `hyprsunset` which is available for hyprland 0.45.0
